src/components/Footer/AletheiaSocialMediaFooter.tsx

```diff
@@ -24,50 +26,20 @@ const AletheiaSocialMediaFooter = () => {
                 </h3>
             </Col>
             <Col span={24}>
-                <SocialIcon
```

Comment:
chore: Create a React component called `AletheiaSocialMediaIcons.tsx` to encapsulate the social media icons code